Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java

```diff
@@ -1,327 +1,292 @@
 package com.example.Project.Resume.Graph.service;
-
-
+import lombok.Data;
 import org.apache.pdfbox.pdmodel.PDDocument;
 import org.fit.pdfdom.PDFDomTree;
 import org.json.JSONObject;
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
 import org.jsoup.nodes.Element;
 import org.jsoup.select.Elements;
-
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import org.springframework.web.multipart.MultipartFile;
-
 import java.io.*;
-
 import java.util.ArrayList;
-
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 @Service
+@Data
 public class ReadPdfService {
-
     @Autowired
     FileManageService fileManageService;
-
     @Autowired
     JsonStringService jsonStringService;
-
+    int expBegins = 0;
```

Comment:
what are these variables? are you using all these variables?

Reply:
yes, they are all in use